webpack/assets/javascripts/react_app/constants.js

```diff
@@ -1,5 +1,27 @@
+// API
 export const STATUS = {
   PENDING: 'PENDING',
   RESOLVED: 'RESOLVED',
   ERROR: 'ERROR',
 };
+
+// Documentation
+export const FOREMAN_VERSION = '1.22';
+export const DOCUMENTATION_URL = `https://theforeman.org/manuals/${FOREMAN_VERSION}/index.html`;
```

Comment:
Since we're not getting props from the server in react routes, are we OK with saving constants in the client? @tbrisker @ohadlevy

Reply:
Can we initialize these consts from the react base layout, so we don't have to maintain them in two different places? Perhaps add some object to redux that stores these and possibly others (foreman `base_url` comes to mind, since we have a couple of bugs related to react not being aware of it)

Reply:
seems possible, I prefer doing it in a future PR, this one is getting too big.

Reply:
Perhaps start splitting this pr into smaller ones that are easier to review and merge?  I highly doubt anyone is capable of properly reviewing 4.5K lines of changes, I know I certainly can't.